contributor/outline.md

```diff
@@ -83,18 +84,19 @@ Outline for this section inspired by the [Trusted Committer introduction video].
   Keep that in mind and plan to start early enough so that your work is available for you to leverage at the time you need it.
     - If the host team is in a different time zone then keep that in mind and plan for it when interacting with them.
     - For larger contributions, get agreement from the host team on the user/technical design of the contribution _before_ submitting a pull request.
-  This type of high-level, up-front agreeemment early will save time in rework or rejection of your pull request down the road.
+  This type of high-level, up-front agreement early will save time in rework or rejection of your pull request down the road.
     - Understand how the host team would like you to reach out for help and questions while you're working.
   - Development
     - If you get stuck, first try to unblock yourself by reading the project documentation or looking at other parts of the code in the project.
     - After trying the above if you're still stuck then reach out to the host team for help.
-    It doesn't do either side any good for you to spin forever on a problem that could be resolved in a few-minute conversation with someone more knowledgable about the project.
+    It doesn't do either side any good for you to spin forever on a problem that could be resolved in a few-minute conversation with someone more knowledgeable about the project.
     It's OK to ask for help.
-    You aren't expected to be as knowledgable and proficient as you might be on the full-time projects that your team owns.
-    - Match the existing coding style, indentation, etc, even if it's not what you would normally do and even if it is not specified in the `CONTRIBUTING.md`.
-    A guest contribution for a feature is not the time to introduce a new way of structuring or formatting project code.
+    You aren't expected to be as knowledgeable and proficient as you might be on the full-time projects that your team owns.
+    - Match the existing coding style, indentation, etc., even if it's not what you would normally do and even if it is not specified in the `CONTRIBUTING.md`.
+    A guest contribution for a feature or bug fix is not the time to introduce a new way of structuring or formatting project code.
   - Submitting the pull request
-    - Run the project build and automated validation before opening a pull request.
+    - Add tests for your code contributions to help making the code maintainable over time.
```

Comment:
Does this point belong here as blanket advice that we give to _all_ contributors 

\- or -

Does this point belong in a project's `CONTRIBUTING.md` (which this outline already references)

?

Reply:
I think that might actually be 'blanket advice' grade. 
Is there a realistic chance that any project could not want tested code?

Reply:
I was thinking of textual contributions, but you're right it does say `code` contributions right there.  Sure, makes sense.

Reply:
My motivation here is that this is a significant part of the contribution that may be overlooked specifically by less experienced people or people with little to no experience with open source.